### DISCUSSION TITLE
New maintainer for Ukrainian

### DISCUSSION THREAD

**Discussion by sircharlo** [sircharlo/meeting-media-manager#1024] [Translations]
Welcome @mchaplyak to the team! @mtdvlpr , can you add him to the docs GitLocalize site?

**Comment by mtdvlpr** [comment]
@sircharlo, from now on you can do so as well :)

**Reply by sircharlo** [reply]
Oh nice! Thanks 😃

**Comment by mchaplyak** [comment]
Thank you! Glad to help :)

**Reply by mtdvlpr** [reply]
Thank you for your willing spirit!

**Comment by mtdvlpr** [comment]
@mchaplyak, I may have cleared your translation progress accidentally, when adding your language to the language list. If that was the case, I apologize!

**Reply by mchaplyak** [reply]
It's not a big deal 😄
